CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+- [Sugerir novo recurso](https://github.com/filipedeschamps/tabnews.com.br/issues/new?assignees=&labels=novo+recurso&projects=&template=2_feature_request.yml): para sugerir um novo recurso, funcionalidade ou melhoria para o TabNews.
+- [Faça uma pergunta](https://github.com/filipedeschamps/tabnews.com.br/issues/new?assignees=&labels=d%C3%BAvida&projects=&template=3_question.yml): para você tirar alguma dúvida sobre o repositório, código ou implementação de algo do TabNews.
+
+Antes de criar um novo issue, [verifique se ele já existe](https://github.com/filipedeschamps/tabnews.com.br/issues) e complemente a discussão com o seu ponto de vista ou com uma sugestão de implementação, se necessário. O título, descrição e comentários em issues devem ser feitos em português.
+
+## Participar de revisões de Pull Requests (PRs)
+
+Mesmo não sendo um mantenedor do repositório, você também pode revisar os pull requests, apontando erros que encontrou enquanto lia o código ou testava a implementação. Isso ajudará quem criou o PR e a pessoa que for avaliar o código antes de realizar o merge, possibilitando um processo de integração mais rápido.
+
+Se você não possui certeza sobre algo, deixe claro no seu comentário para que um mantenedor possa responder suas dúvidas.
+
+## Enviar PRs com soluções previamente debatidas
+
+Se o problema que você quer resolver ainda não estiver documentado em um issue, então [leia o tópico anterior](#participar-de-debates-em-issues-do-repositório) e primeiro exponha o problema, depois proponha a solução (no próprio issue e sem preocupação com a implementação). Isso evitará que você invista seu tempo realizando uma modificação no código que não será aceita por não ser algo desejado ou que o comportamento esperado ainda não foi bem definido.
+
+Se você está procurando algo para desenvolver como sendo a sua primeira interação com o código do repositório, você pode procurar por [issues com o label _"good first issue"_](https://github.com/filipedeschamps/tabnews.com.br/contribute), que são tarefas mais fáceis e simples, ideais para quem nunca interagiu com o código do TabNews ou quem nunca fez uma contribuição para um projeto de código aberto.
+
+As alterações no código devem estar em inglês (nomes de funções, variáveis etc.) e seguir o padrão do projeto. Para entender como rodar o projeto e realizar suas alterações, leia as seções relacionadas no [README](/README.md#instalar-e-rodar-o-projeto).
+
+Após realizar as alterações, você pode [criar um novo Pull Request](https://github.com/filipedeschamps/tabnews.com.br/compare). A descrição estará pré-preenchida com comentários, que servem para te guiar a criar a descrição adequada, contendo as modificações realizadas no código e qual o impacto delas. Isso irá facilitar a revisão do PR por colaboradores do repositório. O título e a descrição do PR devem estar em português, e os commits em inglês.
```

Comment:
Deixando meus 2 centavos aqui, seria bom ter nesse trecho instruções de como é feito uma contribuição em um projeto open source, fazer o fork do projeto, clonar localmente, etc.

Ou então colocar links que tenham essas instruções, por exemplo essas duas documentações do próprio GitHub, um ensina [como contribuir para um projeto Open Source](https://opensource.guide/pt/how-to-contribute/) e tem as instruções de [como abrir um PR](https://opensource.guide/pt/how-to-contribute/#abrindo-um-pull-request) e outro de [como manter o fork sincronizado](https://docs.github.com/pt/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) com as alterações do repositório original

Reply:
@diegosano, como não é algo específico do TabNews, isso foge do foco do documento.

Nó máximo colocaria os links recomendados, mas ficaria como uma recomendação pessoal de quem está escrevendo o documento.

Reply:
Pensei em adicionar no final da linha 42:

```
Para mais detalhes sobre a criação de um pull request, consulte a [documentação do GitHub](https://docs.github.com/pt/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
```

Que tal?

Não parece ter um link na documentação com todas as informações, mas a partir desse link o leitor pode ir para os outros citados, conforme ter dúvidas sobre o processo. Eu achei o texto em português bem confuso por causa dos termos, mas pensei em usar ele pelo repositório ser em português, e quem quiser pode mudar o texto da documentação para inglês.